cmake/pika_add_test.cmake

```diff
@@ -54,10 +54,16 @@ function(pika_add_test category name)
     elseif(${name}_THREADS EQUAL -2)
       set(${name}_THREADS "cores")
       set(run_serial TRUE)
+    elseif(${name}_THREADS EQUAL -3)
+      set(${name}_THREADS "default")
+      set(run_serial TRUE)
     endif()
   endif()
```

Comment:
Maybe these magic numbers could be replaced by the actual names of the command line options?